iog-api/iog_api/models.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+
+from db import Base
+
+
+class Schemas(Base):
+    __tablename__ = 'schemas'
+    id = Column(Integer, primary_key=True)
+    name = Column(String, unique=True)
+    schema = Column(String)
```

Comment:
Is it worth making the Column the JSON type?
https://docs.sqlalchemy.org/en/20/dialects/sqlite.html